mitmproxy/addons/tutorialapp/templates/mitmproxy/index.html

```diff
@@ -0,0 +1,210 @@
+            We first need to understand how the application works.
+            So let's start with investigating what happens when we submit a vote.
+        </p> -->
+
+        <h2 id="view-flows" class="pt-3 pb-3 mb-3 border-bottom">mitmproxy's User Interface</h2>
+
+        <!-- <p>We suggest that you position the windows next to ...</p> -->
+
+        <p>
+            First of all,we needs to get familiar with mitmproxy's user interface.
+            Open the terminal window in which you started mitmproxy.
+            You should see the <span class="badge badge-info">Default View</span> of mitmproxy, which shows a table-like list of flows.
+            Every line represents a request and (optionally) its response using the following columns:
+        </p>
+        <table class="table table-sm">
+            <thead>
+              <tr>
+                <th scope="col">Time</th>
+                <th scope="col">Protocol</th>
+                <th scope="col">Method</th>
+                <th scope="col">Host</th>
+                <th scope="col">Path</th>
+                <th scope="col">Response code</th>
+                <th scope="col">Response size</th>
+                <th scope="col">Response time</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td><code>&gt;&gt;</code> 18:37:23</td>
+                <td>HTTP</td>
+                <td>GET</td>
+                <td>tutorial.mitm.it</td>
+                <td>/</td>
+                <td>200</td>
+                <td>3k</td>
+                <td>179ms</td>
+              </tr>
+            </tbody>
+          </table>
+        <p>
+            <!-- The columns show the date, protocol, method, URL of the flow, among other details on the right. -->
+            In your started instance of mitmproxy you see your browser's HTTP requests to load this tutorial.
+            The <code>&gt;&gt;</code> next to the first flow on the right denotes the currently focussed flow - more on that below.
+            mitmproxy adds rows to the view as new requests come in. Let's try that out:
+        </p>
+
+        <ol>
+            <li>Use the voting app on the right to generate a vote for Lucy.</li>
+            <li>Watch how mitmproxy creates a new line for every vote you submit.</li>
+        </ol>
+
+        <figure class="figure">
+            <img src="/static/images/mitmproxy_flow_list.png" alt="..." class="figure-img img-fluid rounded">
+            <figcaption class="figure-caption text-center">mitmproxy's <span class="badge badge-info">Default View</span></figcaption>
+        </figure>
+
+        <p>
+            mitmproxy is controlled using keyboard shortcuts.
+            Every shortcut is internally bound to a command.
+            Advanced users can easily re-map shortcuts using the configuration file.
+            For now, let's focus on some basic shortcuts:
+        </p>
```

Comment:
Let's go from concrete to the abstract: We can cut this paragraph and move the re-map part either to the end of the section, to "What's next?", or omit it entirely.

Reply:
Fully agree, its a bit "thrown in" there and breaks the didactic "flow".